### DISCUSSION TITLE
The Filter Tags have been operational in the other DevOps demo correct?

### DISCUSSION THREAD

**Discussion by 1bentley** [leightonavantgardehaus/.github#13] [Q&A] [answered]
Am I doing something wrong with filters? Would you have an alternative perhaps on how to trigger a workflow only when a new tag is created?

**Comment by TL8N** [comment, marked as answer]
I suspect whats happening here is that its triggering everything on branches: main OR tags: 'v*' which is not the same as other things - which is awful. a workaround is to use something like if: github.ref == 'refs/heads/whatever'

The second DevOp demo from Monday performed exceptionally.
